src/main/scala/io/druid/indexer/spark/SparkBatchIndexTask.scala

```diff
@@ -259,6 +261,8 @@ object SparkBatchIndexTask
   private val DEFAULT_TARGET_PARTITION_SIZE: Long   = 5000000L
   private val CHILD_PROPERTY_PREFIX        : String = "druid.indexer.fork.property."
   val log            = new Logger(SparkBatchIndexTask.getClass)
+  val lifecycle      = SerializedJsonStatic.lifecycle
```

Comment:
I believe lifecycle needs to be started in order to use an emitter. I tested it without starting lifecycle explicitly, and the emitter failed to `emit()` due to `Service Closed` error. 

Also, from Druid code, all the `ServerRunnable`'s call `initLifecycle()` to start the lifecycle. So I think it is ok to start and close it here.